php-classes/Slate/CBL/Tasks/Attachments/GoogleDocument.php

```diff
@@ -2,16 +2,33 @@
 
 namespace Slate\CBL\Tasks\Attachments;
 
-class GoogleDocument extends AbstractTaskAttachment
+class GoogleDocument extends Link
```

Comment:
@nbey why should this extend link? A google drive file doesn't have a URL field to store distinctly from its ID. It seems to me that it should continue extending `AbstractTaskAttachment`